build_tools/aws-sdk-code-generator/templates/apig_endpoint_class.mustache

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
If you could, please insert a newline after all of these.

Reply:
Done!